src/Symfony/Component/Validator/Constraints/Length.php

```diff
@@ -49,13 +59,17 @@ class Length extends Constraint
     public $min;
     public $charset = 'UTF-8';
     public $normalizer;
+    /* @var self::COUNT_* */
+    public $countUnit = self::COUNT_CODEPOINTS;
```

Comment:
I suggest using a typed property here (we cannot easily add types for existing public properties as that would be a BC break, but new ones should be typed IMO)

Reply:
gladly ;)